index.html

```diff
@@ -6,23 +6,19 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
     <link rel="stylesheet" href="styles.css">
-</head>
-<body>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+<body id="body">
    
     <header id="nav-bar">
         <a href="#" id="name">
                Shoaib</a>
                 <ul id="header-desktop">
            <li class="nav-links"><a id="portfolio" href="#home-2">Portoflio</a></li>
-            <li  class="nav-links"><a id="about-header" href="#about-me">About</a></li>
-            <li  class="nav-links"><a id="contact-header" href="#form">Contact</a></li>
-            <li  id="letter-icon"><a href="#"><img src="images/letter-icon.svg" id="letter-icon"></a></li>
+            <li class="nav-links"><a id="about-header" href="#about-me">About</a></li>
+            <li class="nav-links"><a id="contact-header" href="#form">Contact</a></li>
+            <li id="letter-icon"><a href="#"><img src="images/letter-icon.svg" id="letter-icon"></a></li>
         </ul>
-            <div class="hamburger" id="hamburger">
-            <span class="bar"></span>
-            <span class="bar"></span>
-            <span class="bar"></span>
-        </div>
+        <span class="material-symbols-outlined"style="font-size: 36px;" id="menu-1">menu</span>
```

Comment:
- [ ] Good Job for Implementing all the functionality But You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). So Kindly change "Menu" and "Close" button with `hamburger` and `close` button Just Like the Figma design.

Your Project Mobile Menu Look likes this 
![Capture](https://user-images.githubusercontent.com/93598189/203502715-de8ddc89-ade7-4e0b-86c7-b135750a3e55.PNG)

![Capture](https://user-images.githubusercontent.com/93598189/203502804-458318e8-aeb9-4e6f-a41f-415a8353cbfa.PNG)

The Figma design Look like this : 

![Capture](https://user-images.githubusercontent.com/93598189/203503056-ac620f70-35c8-463f-a78f-65c506c4725a.PNG)

![Capture](https://user-images.githubusercontent.com/93598189/203503140-ba92636c-1a47-4f6a-a3a7-535a356ae4b9.PNG)